### PR TITLE
Rev go build to pick up go 1.8.1

### PR DESCRIPTION
**Pull request by fasaxc**

### CODE REVIEW

**Comment by fasaxc** [comment]
retest this please

**Comment by fasaxc** [comment]
retest this please

File: calc/async_calc_graph.go [hunk 1]
```diff
@@ -187,6 +187,7 @@ func (acg *AsyncCalcGraph) onEvent(event interface{}) {
 
 func (acg *AsyncCalcGraph) Start() {
 	log.Info("Starting AsyncCalcGraph")
-	acg.flushTicks = time.Tick(tickInterval)
+	flushTicker := time.NewTicker(tickInterval)
+	acg.flushTicks = flushTicker.C
```
Comment (neiljerram) [inline calc/async_calc_graph.go]: 1. After pondering our CI failures, I believe this should be `NewTicker`, not `NewTimer`.

2. What motivated this change anyway?

Comment (neiljerram) [inline calc/async_calc_graph.go]: Oh, you just fixed it... :-)

Comment (fasaxc) [inline calc/async_calc_graph.go]: It was a linter false positive.

**Pull request merged**